.github/workflows/make-zips.yml

```diff
@@ -0,0 +1,94 @@
+name: Make Zips CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        platform: ["zynq", "zynqmp"]
+
+    runs-on: ubuntu-latest
+    steps:
+      
+      # Git repositories
+      - name: Checkout Source
+        uses: actions/checkout@v2
+        with:
+          path: PandABlocks-rootfs
+
+      - name: Checkout rootfs builder
+        uses: actions/checkout@v2
+        with:
+          repository: dls-controls/rootfs
+          path: rootfs
+
+      # ARM GNU Toolchain
+      - name: Download Arm GNU toolchain - ${{ matrix.platform }}
+        run: bash PandABlocks-rootfs/.github/scripts/GNU-toolchain.sh ${{ matrix.platform }}
+
+      # Tar dependancies
+      - name: Download tar file dependencies
+        run: bash PandABlocks-rootfs/.github/scripts/tar-files.sh
+
+      # System dependencies
+      - name: Install system dependencies
+        run: |
+          sudo apt-get install -y libncurses5-dev expat
+          sudo pip install rst2pdf
+
+      #CONFIG files
+      - name: Generate CONFIG files - ${{ matrix.platform }}
+        run: |
+          mkdir build
+          bash PandABlocks-rootfs/.github/scripts/config-file-pbrootfs.sh ${{ matrix.platform }}
+          bash PandABlocks-rootfs/.github/scripts/config-file-rootfs.sh
+
+      # Verification of files/file structure
+      - name: List workspace files
+        run: |
+          ls ${{ github.workspace }}
+          echo " "
+          ls PandABlocks-rootfs
+          echo " "
+          ls rootfs
+          echo " "
+          ls tar-files
+          echo " "
+          cat PandABlocks-rootfs/CONFIG
+          echo " "
+          cat rootfs/CONFIG.local
+
+      # Build
+      - name: Make zips
+        run: |
+          cd PandABlocks-rootfs
+          make zips
+
+      # Verification of build files
+      #- name: List build files
+      #  run: ls build
+
+      # Artifacts
+      - name: Upload deps and boot build files
+        uses: actions/upload-artifact@v2
+        with:
+          name: build-files-${{ matrix.platform }}
+          path: |
+            build/deps-*.zip
+            build/boot-*.zip
+
+      # Release 
+      - name: Release
```

Comment:
This will make 2 releases, do something like: https://github.com/dls-controls/scanspec/blob/91dbc3c098e85675fdc3682422836afe594b0906/.github/workflows/code.yml#L64-L89